src/main/java/sigma/TaskList.java

```diff
@@ -0,0 +1,54 @@
+package sigma;
+
+import sigma.task.Task;
+
+import java.util.ArrayList;
+
+/**
+ * Represents a list of tasks.
+ * Contains operations to add, remove, and get tasks.
+ * Also contains operations to check if the list is empty and to get the size of the list.
+ */
+public class TaskList {
```

Comment:
Well-defined `TaskList` class! Just remember to add JavaDoc header comments

Reply:
Alright!